TUM Campus App/TUFilmCardCell.swift

```diff
@@ -14,8 +14,8 @@ class TUFilmCardCell: CardTableViewCell {
     override func setElement(_ element: DataElement) {
         if let unwrappedMovie = element as? Movie{
             titleLabel.text = unwrappedMovie.text
-            let cal = Calendar.current
-            let components = (cal as NSCalendar).components([NSCalendar.Unit.month, NSCalendar.Unit.day], from: unwrappedMovie.airDate as Date)
+            let calendar = Calendar.current
+            let components = calendar.dateComponents([.month, .day], from: unwrappedMovie.airDate as Date)
```

Comment:
I'm not yet sure, if it's really necessary. It's just the update to Swift 3: http://stackoverflow.com/a/38933800/2977288